pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -0,0 +1,64 @@
+package ossaccesscontrol
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/registry"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/accesscontrol/evaluator"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+// OSSAccessControlService is the service implementing role based access control.
+type OSSAccessControlService struct {
+	Cfg               *setting.Cfg `inject:""`
+	Log               log.Logger
+	builtInRoleGrants map[string][]string
+}
+
+func init() {
+	registry.RegisterService(&OSSAccessControlService{})
+}
+
+// Init initializes the OSSAccessControlService.
+func (ac *OSSAccessControlService) Init() error {
+	ac.Log = log.New("accesscontrol")
+	ac.builtInRoleGrants = builtInRoleGrants
+
+	return nil
+}
+
+func (ac *OSSAccessControlService) IsDisabled() bool {
+	if ac.Cfg == nil {
+		return true
```

Comment:
Shouldn't we log an error if Cfg has not been injected?

Reply:
It's only for tests (Cfg always injected in the real life).